draft-packages/select/KaizenDraft/Select/README.mdx

```diff
@@ -82,6 +82,10 @@ Use a pre-selected good default where possible. In some cases, it cuts down on u
 *   **[Focus indicator](/guidelines/interaction-states#focus-indicator):**
       *   On keyboard navigation and click show focus indicator on the select field.
       *   For select options show focus indicator on keyboard navigation and on click (not hover).
+ *   **[Autocomplete](/components/autocomplete):**
+      *   Base select field should indicate that it is a searchable select, either through hint text, a search icon, or both.
+      *   When the user starts typing, autocomplete should show a list of options in the dropdown that match the field input.
+      *   A selection is not made until the user actively makes one, either by choosing an autocompleted suggestion from the select, or by finishing typing a matched option in the field and then selecting enter.
```

Comment:
Should we clarify what happens if they select enter without a matched option? (I assume nothing)